### ISSUE TITLE
css-what dependency is vulnerable to Denial of Service

### ISSUE THREAD

**Issue by IlyaShestakov** [ben-eb/gulp-svgmin#114]
When using gulp-svgmin@3.0.0 bpm audit reports
```
  High            Denial of Service
  Package         css-what
  Patched in      >=5.0.1
  Dependency of   gulp-svgmin [dev]
  Path            gulp-svgmin > svgo > css-select > css-what
  More info       https://npmjs.com/advisories/1754
```

svgo issue to upgrade dependency: https://github.com/svg/svgo/issues/1488

**Comment by rejas** [comment]
Thanks for the report. Preparing a v4 release but would like to wait for https://github.com/svg/svgo/pull/1485 to get merged into svgo and released so that we can get rid of the warning.

**Comment by rejas** [comment]
Fixed with the v4.0.0 release.
